### PR TITLE
Update test related to integer subtypes

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/4543
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

**Review by dilanSachi — APPROVED** [review]
LGTM.
Just to confirm, `int32`, `int64`, `uint64`, `sint64`, `fixed32`, `fixed64` are still considered as int because we dont have lang support for them right?

File: ballerina-tests/tests/76_number_types_pb.bal [hunk 1]
```diff
@@ -15,10 +15,11 @@
 // under the License.
 
 import ballerina/grpc;
+import ballerina/lang.'int;
```
Comment (dilanSachi) [inline ballerina-tests/tests/76_number_types_pb.bal:18]: btw, why do we need this import?
We can directly do `int:Unsigned32` right?

**Comment by MadhukaHarith92** [comment]
> LGTM. Just to confirm, `int32`, `int64`, `uint64`, `sint64`, `fixed32`, `fixed64` are still considered as int because we dont have lang support for them right?

Yes, as per the current spec﻿[[1]](﻿﻿https://lib.ballerina.io/ballerina/lang.int/2201.7.0) those aren't supported.

[1] https://lib.ballerina.io/ballerina/lang.int/2201.7.0

File: ballerina-tests/tests/76_number_types_pb.bal (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina-tests/tests/76_number_types_pb.bal]: Removed in [4d4df2d](https://github.com/ballerina-platform/module-ballerina-grpc/pull/1351/commits/4d4df2db6c2ddf1ecdf4f9f770273ad7dc2e0961)

**Pull request merged**